### ISSUE TITLE
[Accessibility] Some words on the "No Internet" error page are split across two or three lines on the iPad Mini

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#23915]
**Steps to reproduce:**

1. Turn dynamic text on the largest scale.
1. Open a website on iPad Mini in portrait mode.
1. Turn off the Wi-Fi connection and then reload the page.
1. Observe the behavior.

**Actual results**

Some words on the "No Internet" error page are split across two or three lines on the iPad Mini

**Expected results**

All labels on the "No Internet" error page are written on a single line, ensuring proper alignment and readability.

**Device & build information**

* Device:  iPad mini A17 Pro (18.0)
* Build version: v9000(49218)
* First seen version: ?

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-10947)

**Comment by anna-yeddi** [comment]
thank you for reporting this bug! While it is not a technical failure for the WCAG 2.x [Level AA 1.4.4 Resize Text](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html) and/or [Level AA 1.4.10 Reflow](https://www.w3.org/WAI/WCAG22/Understanding/reflow.html) success criteria, making sure all words are not split would allow for better readability for Fx users.

**Comment by lmarceau** [comment]
This is what the error page should look like on iPhone according to design:

![Image](https://github.com/user-attachments/assets/3b636778-3bc0-410a-8321-d1566825410c)

**Comment by data-sync-user** [comment]
➤ Yoana Rios Diaz commented:

For QA: The fix for this case is to use portrait layout (meaning all the content is shown vertically) when dynamic text is using large scale to use all the width of the screen to show the text

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v145 (63787) with iPad mini 5 (16.7.2)

  * !image-20251105-140933.png|width=392,alt="image-20251105-140933.png"!

!image-20251105-140923.png|width=388,alt="image-20251105-140923.png"!
